op-node/node/node.go

```diff
@@ -197,9 +200,8 @@ func (n *OpNode) initRuntimeConfig(ctx context.Context, cfg *Config) error {
 			return l1Head, err
 		}
 
-		n.handleProtocolVersionsUpdate(ctx)
-
-		return l1Head, nil
+		err = n.handleProtocolVersionsUpdate(ctx)
+		return l1Head, err
```

Comment:
Is there a reason to return the `l1Head` on error? It seems more idiomatic (according to golang) to do the following.
```suggestion
		if err := n.handleProtocolVersionsUpdate(ctx); err != nil {
		    return eth.L1BlockRef{}, err
		}

		return l1Head, nil
```

Also asking because above, the `l1Head` is still returned when the `runCfg.Load` call errors on line 200.

Reply:
Yeah. It's not pretty, but the reason is so we can log the last protocol version update on L230 even when there's a halt error.

Reply:
Got it makes sense!